pkg/deploy/lattice/target_group_synthesizer.go

```diff
@@ -38,38 +38,31 @@ type targetGroupSynthesizer struct {
 }
 
 func (t *targetGroupSynthesizer) Synthesize(ctx context.Context) error {
```

Comment:
Please double check if this is impacting `serviceexport` .   `serviceexport` creation and deletion need this one to create or delete target groups.

Reply:
Tested the noral serviceexport creation and deletion use cases, it could create and delete the target group